app/scripts/components/stories/hub/index.tsx

```diff
@@ -161,150 +163,153 @@ function StoriesHub() {
     <PageMainContent>
       <LayoutProps
         title={getString('stories').other}
-        description={getString('storiesBanner')?.other || 'Explore the guided narratives below to discover how NASA satellites and other Earth observing resources reveal a changing planet.'}
-      />
-      <PageHero
-        title={getString('stories').other}
-        description={getString('storiesBanner')?.other || 'Explore the guided narratives below to discover how NASA satellites and other Earth observing resources reveal a changing planet.'}
+        description={getString('storiesBanner').other}
       />
-
+      <ComponentOverride with='storiesHubHero'>
+        <PageHero
+          title={getString('stories').other}
+          description={getString('storiesBanner').other}
```

Comment:
💭 : Future case - not a blocker. This makes me think we should probably validate the config if this isn't happening already somewhere? To make sure required keys in the config exist otherwise optional chaining can help here

Reply:
@danielfdsilva 's comment probably explained the current state of the return value of `getString.` I just want to mention that I also thought our type was wrong and tried to fix it in this pr: (reverted it back.) https://github.com/NASA-IMPACT/veda-ui/pull/735

So my 💭 for the future is `getString` is trying to do too many things, and makes the use of it confusing. It might be worth separating the data that needs `one`, `other` from the one that doesn't need it. But I also think this is out of the scope of this PR.